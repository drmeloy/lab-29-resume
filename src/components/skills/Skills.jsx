import React from 'react';
import PropTypes from 'prop-types';

const Skills = ({ skills }) => {  
  const skillsDisplay = skills.map((skill, i) => (
    <li key={i}>skill</li>
  ));

  return(
    <>
      <h2>Skills</h2>
      <ul>{skillsDisplay}</ul>
    </>
  );
};

Skills.propTypes = {
  skills: PropTypes.array.isRequired
}

export default Skills;
