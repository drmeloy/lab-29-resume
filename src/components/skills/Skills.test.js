import React from 'react';
import { shallow } from 'enzyme';
import Skills from './Skills.jsx';

describe('Skills component', () => {
  it('renders a Skills component', () => {
    const wrapper = shallow(<Skills skills={[]} />);
    expect(wrapper).toMatchSnapshot();
  })
});
