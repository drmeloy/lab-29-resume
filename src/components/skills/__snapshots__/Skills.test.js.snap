// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Skills component renders a Skills component 1`] = `
<Fragment>
  <h2>
    Skills
  </h2>
  <ul />
</Fragment>
`;
